tools/clang/unittests/HLSL/PixTest.cpp

```diff
@@ -4020,34 +4029,34 @@ void main()
         auto Testables = TestStructAnnotationCase(hlsl, optimization);
 
         if (!choice.IsOptimized) {
-            VERIFY_ARE_EQUAL(1, Testables.OffsetAndSizes.size());
-            VERIFY_ARE_EQUAL(4, Testables.OffsetAndSizes[0].countOfMembers);
-            VERIFY_ARE_EQUAL(0, Testables.OffsetAndSizes[0].offset);
+            VERIFY_ARE_EQUAL(1u, Testables.OffsetAndSizes.size());
+            VERIFY_ARE_EQUAL(4u, Testables.OffsetAndSizes[0].countOfMembers);
+            VERIFY_ARE_EQUAL(0u, Testables.OffsetAndSizes[0].offset);
             // 8 bytes align for uint64_t:
             VERIFY_ARE_EQUAL(32 + 16 + 16 /*alignment for next field*/ + 32 + 32/*alignment for max align*/ + 64,
```

Comment:
another unsigned literal

Reply:
This looks to be the only one remaining. Did you leave it out intentionally?

Reply:
No. It missed because the (int) is in another line:(

Fixed.